osbs/utils.py

```diff
@@ -411,7 +419,13 @@ class Labels(object):
         LABEL_TYPE_VENDOR: ('vendor', 'Vendor'),
         LABEL_TYPE_SOURCE: ('authoritative-source-url', 'Authoritative_Registry'),
         LABEL_TYPE_COMPONENT: ('com.redhat.component', 'BZComponent'),
-        LABEL_TYPE_HOST: ('com.redhat.build-host', 'Build_Host')
+        LABEL_TYPE_HOST: ('com.redhat.build-host', 'Build_Host'),
+        LABEL_TYPE_RUN: ('run', 'RUN'),
+        LABEL_TYPE_INSTALL: ('install', 'INSTALL'),
+        LABEL_TYPE_UNINSTALL: ('uninstall', 'UNINSTALL')
+    }
+    LABEL_NAMES_EQUAL = {
+        LABEL_TYPE_DESCRIPTION: ('description', 'io.k8s.description')
```

Comment:
Why do we consider those equal? https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/vendor/redhat/labels.md lists both, but doesn't state that any of those can be used